apps/bttester/src/btp_gap.c

```diff
@@ -515,6 +515,10 @@ start_advertising(const void *cmd, uint16_t cmd_len,
 
 #if MYNEWT_VAL(BLE_EXT_ADV)
     adv_params.own_addr_type = own_addr_type;
+
+    if (sd_len != 0) {
```

Comment:
we may also have to unset it so I'd suggest

adv_params.scannable = sd_len != 0;

Reply:
This will need to be further discussed, as there are situations where sd_len is actually 0, but we don't want to set scannable to 0. TBD